### ISSUE TITLE
About Learning Rate Adjustment

### ISSUE THREAD

**Issue by Joker316701882** [ultralytics/yolov5#2688]
## ❔Question

Hi @glenn-jocher 
I'm curious about the learning rate adjustment in this repo (a.k.a. from 0.01 to 0.002 via cos annealing). Commonly, the learning rate is decayed by 100 as in many other repos(mmdet, detectron2, pp-yolo), but the learning rate here is only decayed by 5. 

I conducted a few experiments and found that only when EMA is used, the model can converge well under such lr scheduler(decayed by 5). However, in this case, although EMA model performs well, the original model performs extremely poorly compared to this EMA version. Looks like the EMA is the key fact of this LR scheduler.

So my question is
1).  How did you learn to use EMA with such an uncommon LR scheduler (1e-2->2e-3, decayed by 5)?
2). What if you remove the EMA and use a common LR scheduler(1e-2->4e-2, decayed by 100), how much degradation of mAP in that case?

Thank you.

**Comment by glenn-jocher** [comment]
@Joker316701882 yes the LR, schedule and EMA are all interrelated, so the current hyps (lr0, lrf) were optimized to work well with the EMA. If we lower the lrf too low though, to 0.01 as in your example) then we see earlier overfitting and lower best mAP.

I think the inclusion of the EMA helps beyond what any non-EMA model could be trained to, though it would be interesting to see your experiment results of training without it to a much lower lrf.

**Comment by Joker316701882** [comment]
Thank you for your reply. I manage to overcome the overfitting by adding more augmentation and successfully trained the model with a lower lrf (i.e., 0.01). But the best mAP is comparable to this repo. I'll keep conducting new experiments and update results here : ).

**Comment by glenn-jocher** [comment]
@Joker316701882 hmm interesting. Yes it's a tricky interplay between the base model, the EMA, and the parameters to suit each best. Yes keep us updated!
